### ISSUE TITLE
Transcribe dataframes to validate table extraction

### ISSUE THREAD

**Issue by katie-lamb** [catalyst-cooperative/mozilla-sec-eia#39]
After LayoutLM labels entities for each word in an Ex. 21 image, we take those entity labels and use a table extraction model to create a structured table of the Ex. 21 table. In order to validate the table extraction model, we need to transcribe a set of Ex. 21 tables so that we can check if the extracted table matches the manually created ground truth table. To transcribe dataframes, follow the instructions on the first sheet in the [Ex. 21 Validation Data spreadsheet](https://docs.google.com/spreadsheets/d/1mMl1B6pRbRMwpjEjFaFCxcZZsCRHMDrOTaCDKkGC4jY/edit?gid=252210337#gid=252210337) which will guide you through transcribing a table and logging its layout. The goal is to have maybe 100 tables transcribed after this first round of transcription.

Also, look at the CorpWatch database to see if some of that data can be used for validation.

Question:
* How much filing overlap should we have between these transcribed tables and the training data that's used for LayoutLM?
